src/org/traccar/database/DeviceManager.java

```diff
@@ -52,6 +55,13 @@
 
     private final Map<Long, Position> positions = new ConcurrentHashMap<>();
 
+    private final Map<Long, Boolean> motionStates = new ConcurrentHashMap<>();
```

Comment:
Do you think it's a good idea to create some "device state" class to store last position, motion status, and possibly some other information in future?